### DISCUSSION TITLE
내 상태와 캐릭터가 조회되지 않는 현상이 있었어요.

### DISCUSSION THREAD

**Discussion by jupapa** [metamong-friends/metamong-friends.github.io#55] [제보하기]
접속되어 있는 상태에서 VDI에서 작업을 30분 정도 하다가 돌아 왔더니 그림과 같이 캐릭터가 조회되지 않는 현상이 있었어요. 다른 메뉴들도 안뜨고요. 종료했다가 다시 접속하니까 현상은 해결되었습니다. 

<img width="1145" alt="메타몽" src="https://github.com/metamong-friends/metamong-friends.github.io/assets/27798291/708331ac-3f05-4fe2-a954-e4cf7a5eeebf">

**Comment by ACT-TJ** [comment]
안녕하세요!

우선. 바쁘신데 제보 감사합니다.

저희가 FailOver 정책을 정리중인데요! 네트워크 이슈, 시스템의 문제 등 연결은 잠시 끊어지는 경우가 있을 수 있다는
결론을 내렸는데 보시는 화면과 같이 사용을 이어갈 수 없는 경우를 없애고자 처리 후 6월 11일 추가 릴리즈를 진행하였고,
Reconnect와 Leave Space 중 선택할 수 있는 팝업을 띄우고 Reconnect 시 재연결을 처리하도록 하였습니다.

시스템 문제로 일시적으로 연결이 끊어지는 부분도 좀 더 원인을 파악해 안정화 할 수 있도록 계속 보완해 가겠습니다.

감사합니다!!
